docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Destination type - Email
+nav_order: 1
+parent: monitors
+has_children: false
+---
+
+#  Email as a Destination 
+A destination is a location where you can send an alert that was triggered by the monitor. To send the alert as an email, select **Email** as the destination **Type**. You would need to configure at least one  **Sender** email address and add a **Recipient** email address. These two settings can be configured and managed using **Manage senders** and **Manage email groups**. 
+
+## Manage Senders
+Manage senders allows you to configure and manage **Sender** email addresses. Unless at least one sender is configured in the settings, the **Sender** dropdown list will be empty. 
+
+To configure a sender email, do the following.
+
+1. Choose **Manage senders**.
+1. In the **Manage email senders** modal window, choose **Add sender**. 
+1. Next, choose **New sender**. Multiple senders can be added one at a time.
+1. Enter a **Sender name**. Duplicate names are not allowed. You must enter a unique and descriptive name by using valid symbols, numbers, or words. A unique and descriptive name is also helpful for search queries.
```

Comment:
Seems excessive. You can just say "Enter a unique name for the sender."

Reply:
done